.gitignore

```diff
@@ -1 +1,2 @@
 /.idea
+/.DS_Store
```

Comment:
ooi, what is this?

Reply:
Its a developer environment artifact I added to ignore list, Happy to take on board better way to handle this ?

Reply:
I guess I should have chimed in sooner, but I'm fine with gratuitous gitignore files.